src/main/java/uk/gov/justice/digital/delius/transformers/AppointmentTransformer.java

```diff
@@ -48,6 +51,15 @@ public static AppointmentDetail appointmentDetailOf(Contact contact) {
             .sensitive(ynToBoolean(contact.getSensitive()))
             .outcome(appointmentOutcomeOf(contact))
             .rarActivity(ynToBoolean(contact.getRarActivity()))
+            .rarRequirementId(
+                Optional.ofNullable(contact.getNsi())
+                    .map(Nsi::getRqmnt)
```

Comment:
Do you need to filter out inactive and soft deleted requirements as both features seem to be used. They were a problem for R&M but may not be for your context.